challenges/08-coding-interview-prep/rosetta-code.json

```diff
@@ -11596,6 +11596,844 @@
+            "assert(Array.isArray(partitions(2,0,2)),'<code>partitions(2,0,2)</code> should return a array.');"
+        },
+        {
+          "text":
+            "'<code>partitions(2,0,2)</code> should return <code>'+JSON.stringify(results[0])+'</code>.'",
+          "testString":
+            "assert.deepEqual(partitions(2,0,2),results[0],'<code>partitions(2,0,2)</code> should return <code>'+JSON.stringify(results[0])+'</code>.');"
+        },
+        {
+          "text":
+            "'<code>partitions(1,1,1)</code> should return <code>'+JSON.stringify(results[1])+'</code>.'",
+          "testString":
+            "assert.deepEqual(partitions(1,1,1),results[1],'<code>partitions(1,1,1)</code> should return <code>'+JSON.stringify(results[1])+'</code>.');"
+        },
+        {
+          "text":
+            "'<code>partitions(1,0,1)</code> should return <code>'+JSON.stringify(results[2])+'</code>.'",
+          "testString":
+            "assert.deepEqual(partitions(1,0,1),results[2],'<code>partitions(1,0,1)</code> should return <code>'+JSON.stringify(results[2])+'</code>.');"
+        },
+        {
+          "text":
+            "'<code>partitions(1,0,2)</code> should return <code>'+JSON.stringify(results[3])+'</code>.'",
+          "testString":
+            "assert.deepEqual(partitions(1,0,2),results[3],'<code>partitions(1,0,2)</code> should return <code>'+JSON.stringify(results[3])+'</code>.');"
+        },
+        {
+          "text":
+            "'<code>partitions(2,0,1)</code> should return <code>'+JSON.stringify(results[4])+'</code>.'",
+          "testString":
+            "assert.deepEqual(partitions(2,0,1),results[4],'<code>partitions(2,0,1)</code> should return <code>'+JSON.stringify(results[4])+'</code>.');"
+        }
+      ],
+      "id": "5a23c84252665b21eecc7f4e",
+      "challengeType": 5,
+      "releasedOn": "August 3, 2018",
+      "files": {
+        "indexjs": {
+          "key": "indexjs",
+          "ext": "js",
+          "name": "index",
+          "contents": [
+            "function partitions (a1, a2, a3) {",
+            "  // Good luck!",
+            "}"
+          ],
+          "head": [],
+          "tail": [
+            "var results=[",
+            "[[[1,2],[],[3,4]],[[1,3],[],[2,4]],[[1,4],[],[2,3]],[[2,3],[],[1,4]],[[2,4],[],[1,3]],[[3,4],[],[1,2]]],",
+            "[[[1],[2],[3]],[[1],[3],[2]],[[2],[1],[3]],[[2],[3],[1]],[[3],[1],[2]],[[3],[2],[1]]],",
+            "[[[1],[],[2]],[[2],[],[1]]],",
+            "[[[1],[],[2,3]],[[2],[],[1,3]],[[3],[],[1,2]]],",
+            "[[[1,2],[],[3]],[[1,3],[],[2]],[[2,3],[],[1]]]]"
+          ]
+        }
+      }
+    },
+    {
+      "title": "Ordered words",
+      "description": [
+        "An  <i>ordered word</i>  is a word in which the letters appear in alphabetic order.",
+        "Examples include  <b>abbey</b>  and  <b>dirt</b>.",
+        "Write a function that returns true if given word is ordered. Otherwise returns false"
```

Comment:
"Write a function that returns true if the letters of the given word are in alphabetic order and false if they're not."